docs/antctl.md

```diff
@@ -204,6 +204,16 @@ antctl get appliedtogroup [NAME] [-o yaml]
 antctl get addressgroup [NAME] [-o yaml]
 ```
 
+NetworkPolicy, AppliedToGroup, and AddressGroup also support `sort-by=''` option,
+which can be used to sort these resources on the basis of a particular field. Any
+valid json path can be passed as flag value. If no value is passed it will use a default field to sort results. For NetworkPolicy, the default field is the name of the source NetworkPolicy. For AppliedToGroup and AddressGroup, the default field is the object name(which is a generated UUID).
```

Comment:
+1
You also need an empty space before `(`:
`the object name (which is a generated UUID).`